.husky/pre-commit

```diff
@@ -2,3 +2,4 @@
 . "$(dirname "$0")/_/husky.sh"
 
 ./node_modules/.bin/lint-staged
+robotidy --configure DiscardEmptySections:allow_only_comments=True --spacecount 2 tests/robot-tests/
```

Comment:
Would be better to add this  `package.json` under a `*.robot` wildcard pattern, otherwise it will be ran on the entire directory for every single commit.

Think this would do the trick:

```json
"*.robot": ["robotidy --configure DiscardEmptySections:allow_only_comments=True --spacecount 2"]
```